.phpcs.xml

```diff
@@ -8,22 +8,15 @@
   <exclude-pattern>./vendor</exclude-pattern>
   <!-- TODO: fix excluded files below -->
   <exclude-pattern>./src/wovn_helper.php</exclude-pattern>
```

Comment:
I agree, I have plan for making autoloading and I think it will be best around that time. I played with autoloading before and I figured out it will be most stable if we fix project directory structure, some filenames (as you said) and namespacing.